.circleci/config.yml

```diff
@@ -0,0 +1,81 @@
+version: 2.1
+orbs:
+  node: circleci/node@1.1.6
+
+references:
+  workspace_root: &workspace_root ~/project
+
+  attach_workspace: &attach_workspace
+    attach_workspace:
+      at: *workspace_root
+
+  filters_tags: &filters_tags
+    filters:
+      tags:
+        only: /.*/
+
+jobs:
+  build:
+    executor: node/default
+    steps:
+      - checkout
+      - node/with-cache:
+          cache-key: yarn.lock
+          dir: ~/.cache/yarn
+          steps:
+            - run: yarn --frozen-lockfile --ignore-scripts
+            - run: yarn lerna bootstrap
+            - run: yarn lerna run build --concurrency=2
+      - persist_to_workspace:
+          root: *workspace_root
+          paths: .
+
+  test:
+    executor: node/default
+    steps:
+      - *attach_workspace
+      - run: yarn lint
+      - run: yarn format
```

Comment:
Does CircleCI break the build if any changes are found? If this is handled different than Travis we may be able to remove the `yarn format` command.

Reply:
As I understand it, the clean working directory check has always been an `npm` step (on version) not Travis. As such, we should leave this intact just as before.